packages/toolkit/src/lib/react-query-service/misc/useBlogPosts.ts

```diff
@@ -46,8 +44,14 @@ const formatDate = (dateString: string | undefined) => {
 };
 
 const fetchBlogPosts = async () => {
+  // Fetch the latest release tag
+  const tagsResponse = await axios.get(
+    "https://api.github.com/repos/instill-ai/instill.tech/tags"
+  );
+  const latestTag = tagsResponse.data[0].name;
```

Comment:
Should it be `tagsResponse.data[0]?.name`? In case there are no posts.

Reply:
If that's the case should I then check for the next tag?

Reply:
yeah, ideally `${latestTag ? `?ref=${latestTag}` : ''}`